Makefile

```diff
@@ -155,8 +155,14 @@ endef
 PRELOAD_VARS := LD_PRELOAD DYLD_FORCE_FLAT_NAMESPACE DYLD_INSERT_LIBRARIES
 
 # use a minimal whitelist of safe environment variables
-# HOME is needed for ~/.gitconfig for patch-tool-mxe
```

Comment:
HOME seemed reasonably self-explanatory, I added a link to [Linux From Scratch](http://www.linuxfromscratch.org/lfs/view/stable/chapter04/settingenvironment.html) instead.